packages/ember-glimmer/tests/integration/helpers/log-test.js

```diff
@@ -0,0 +1,60 @@
+import { RenderingTest, moduleFor } from '../../utils/test-case';
+import { get } from 'ember-metal/property_get';
+import Logger from 'ember-metal/logger';
+
+var originalLog, logCalls;
+
+moduleFor('Helpers test: {{log}}', class extends RenderingTest {
```

Comment:
Negative, this is the one exception that we don't have to :smile: We don't know what the semantics for "re-rendering the log" is supposed to be, and didn't feel like locking it down now